spec/models/queue_item_spec.rb

```diff
@@ -40,6 +87,37 @@
       end
     end
 
+    describe 'normalize_positions' do
```

Comment:
This could use a proper description - check this out - http://www.betterspecs.org/#describe